### ISSUE TITLE
 [pre-release 3.4.205] DNT-header

### ISSUE THREAD

**Issue by cqx931** [dhowe/AdNauseam#1258]
No DNT header is being sent for all requests (including ad clicking)

**Comment by dhowe** [comment]
all platforms?

**Comment by cqx931** [comment]
not valid for the latest master branch, close

**Comment by dhowe** [comment]
I see no DNT headers being sent in the current release. Please check / reply

**Comment by cqx931** [comment]
Chrome:
DNT headers work as expected
Firefox:
No DNT header is being sent for all requests (including ad clicking)

**Comment by dhowe** [comment]
Are you looking at the current master, or the last release (I have recently done a fix in master) ??

I want to understand how the code made it into a public release with DNT headers...

**Comment by cqx931** [comment]
ok, yes, there is no DNT header in the current release. 
The issue seems to start from Release v3.4.205, caused by a merge from uBlock.
Since I opened this ticket, so I must have noticed this issue when I did the testing for 3.4.205 pre-release. Then on 22 Oct I said that this issue is not valid any more on master branch... 
However, when I trace this back with the corresponding commit, this is actually not true. So something must went wrong when I recheck this...Might be a mistake of checking the wrong version.

Then for v3.4.4, we only did the basic testing, which doesn't include DNT headers, that's why the issue was still there in the latest public release.

So I think it makes sense to add DNT headers to the basic release testing.
